tools/modulelint.py

```diff
@@ -45,8 +45,21 @@ def setUp(self):
     def testDockerFromBaseruntime(self):
         self.assertTrue(self.dp.check_baseruntime())
 
-    def testDockerRunMicrodnf(self):
-        self.assertTrue(self.dp.check_microdnf())
+    def testDockerNodocs(self):
+        self.assertTrue(self.dp.check_nodocs_flag())
+
+    def testDockerCleanAll(self):
+        self.start()
+        pkg_mgr = "yum"
+        # Detect distro in image
+        distro = self.run("cat /etc/redhat-release").stdout
+        if 'Fedora' in distro:
+            pkg_mgr = "dnf"
+        # Look, whether we have solv files in /var/cache/<pkg_mgr>/*.solv
+        # dnf|yum clean all deletes the file *.solv
+        ret = self.run("ls /var/cache/%s/*.solv" % pkg_mgr, ignore_status=True)
```

Comment:
Igor, does dnf have an API to see how much disk space local metadata of repositories take up? Looking at the cache folder myself, I'm not sure what's the best way to do the test:

```
[root@633f2e1e1602 /]# cd /var/cache/dnf
[root@633f2e1e1602 dnf]# ll
total 69808
-rw-r--r-- 1 root root        2 Jul 28 07:07 expired_repos.json
drwxr-xr-x 3 root root       42 Jul 28 07:07 fedora-6dbd63560daef6bf
-rw-r--r-- 1 root root 40995822 Jul 28 07:07 fedora-filenames.solvx
-rw-r--r-- 1 root root 19888765 Jul 28 07:07 fedora.solv
drwxr-xr-x 3 root root       42 Jul 28 07:06 updates-2854b3113b7a3c6c
-rw-r--r-- 1 root root  6715186 Jul 28 07:06 updates-filenames.solvx
-rw-r--r-- 1 root root   607310 Jul 28 07:06 updates-presto.solvx
-rw-r--r-- 1 root root  1414773 Jul 28 07:06 updates-updateinfo.solvx
-rw-r--r-- 1 root root  1847142 Jul 28 07:06 updates.solv
[root@633f2e1e1602 dnf]# dnf clean all
18 files removed
[root@633f2e1e1602 dnf]# ll
total 4
-rw-r--r-- 1 root root 21 Jul 28 07:07 expired_repos.json
drwxr-xr-x 3 root root 22 Jul 28 07:07 fedora-6dbd63560daef6bf
drwxr-xr-x 3 root root 22 Jul 28 07:07 updates-2854b3113b7a3c6c

```